docs/sandbox/index.md

```
@@ -7,5 +7,9 @@ layout: false
 import Sandbox from '../.vitepress/theme/components/sandbox/Sandbox.vue'
 </script>
 
+<ClientOnly>
```

Markdownlint (0.37.0)

Element: ClientOnly
Inline HTML

(MD033, no-inline-html)